src/codeu/chat/common/ConversationHeader.java

```diff
@@ -54,8 +54,11 @@ public ConversationHeader read(InputStream in) throws IOException {
   public final Uuid owner;
   public final Time creation;
   public final String title;
+  //public final Uuid creator;
 
   public ArrayList<Uuid> members;
+  public ArrayList<Uuid> owners;
+  public ArrayList<Uuid> creator;
```

Comment:
there can only be one creator. This doesn't need to be a list. 
`public final Uuid creator;`